src/Microsoft.DotNet.Interactive/KernelCommandExtensions.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Microsoft.DotNet.Interactive.Commands;
+using Pocket;
+
+namespace Microsoft.DotNet.Interactive;
+
+internal static class KernelCommandExtensions
+{
+    internal static void StampRoutingSlipAndLog(this KernelCommand command, Uri uri)
+    {
+        command.RoutingSlip.Stamp(uri);
+        command.LogRoutingSlipInfo();
```

Comment:
Nitpick: I would tend toward reversing this, e.g. `Log.RoutingSlipInfo(command);`

Reply:
Fixed.